scripts/install-chromedriver.mjs

```diff
@@ -0,0 +1,70 @@
+import _ from 'lodash';
+import { ChromedriverStorageClient } from 'appium-chromedriver';
+import axios from 'axios';
+
+const VERSION_LATEST = 'LATEST';
+
+function getChromedriverUrl() {
+    return process.env.CHROMELABS_URL || 'https://googlechromelabs.github.io'
+}
+
+function getChromedriverVersion() {
+    return process.env.npm_config_chromedriver_version ||
+        process.env.CHROMEDRIVER_VERSION ||
+        VERSION_LATEST;
+}
+
+async function formatCdVersion (ver) {
+    if (_.toUpper(ver) !== VERSION_LATEST) {
+      return ver;
+    }
+    let jsonStr;
+    const url = `${getChromedriverUrl()}/chrome-for-testing/last-known-good-versions.json`;
+    try {
+      jsonStr = (await axios({
+        url,
+        headers: {
+            'user-agent': 'appium',
+            accept: `application/json, */*`,
+          },
+        timeout: 15000,
+        responseType: 'text',
+      })).data;
+    } catch (e) {
+      throw new Error(`Cannot fetch the latest Chromedriver version. ` +
+        `Make sure you can access ${url} from your machine or provide a mirror by setting ` +
+        `a custom value to CHROMELABS_URL enironment variable. Original error: ${err.message}`);
+    }
+
+    let json;
+    try {
+      json = JSON.parse(jsonStr);
+    } catch (e) {
+      const err = /** @type {Error} */ (e);
+      throw new Error(`Storage JSON cannot be parsed. Original error: ${err.message}`);
+    }
+    /**
+     * "timestamp":"2024-10-20T20:09:22.942Z",
+     * "channels":{
+     *    "Stable":{
+     *       "channel":"Stable",
+     *       "version":"130.0.6723.58",
+     *       "revision":"1356013"
+     * ...
+     */
+    if (!json?.channels?.Stable?.version) {
+      throw new Error('The format of the storage JSON is not supported');
+    }
+    return json.channels.Stable.version;
+  }
+
+async function install () {
+  const client = new ChromedriverStorageClient({
+    chromedriverDir: import.meta.dirname,
```

Comment:
it should be possible to provide target folder path via command line args

Reply:
https://github.com/appium/appium-chromium-driver/pull/366/files#diff-34892ec7ee6d58550ec013d634a05a2ff9706f200850a3759508984819db3f81R15